mobile/src/components/uam/groups-table/groups-table.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { FC, useEffect } from 'react';
+import { useWindowDimensions } from 'react-native';
+
+import { Table, Text, View } from '~/components/common/common';
+import { useAppDispatch, useAppSelector } from '~/hooks/hooks';
+import { uamActions } from '~/store/actions';
+
+import { getGroupsColumns, getGroupsRows } from './helpers/helpers';
+import { styles } from './styles';
+
+const GroupsTable: FC = () => {
```

Comment:
Create also uam screen - components/uam.tsx, put groups table in it and add UAM navigation item to drawer